.circleci/config.yml

```diff
@@ -1101,9 +1101,10 @@ workflows:
       - wp-desktop-mac:
           requires:
             - wp-desktop-source
-      - wp-desktop-linux:
-          requires:
-            - wp-desktop-source
+      # Use TeamCity to run Linux build
```

Comment:
Note this disables the linux build for merges to master. Building releases uses a different workflow and remains unchanged.